packages/amplify_api/example/lib/main.dart

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'dart:async';
+import 'package:amplify_api/amplify_api.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'amplifyconfiguration.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  String _result = '';
+  bool _isAmplifyConfigured = false;
+  Amplify amplify = new Amplify();
+
+  @override
+  void initState() {
+    super.initState();
+    initPlatformState();
+  }
+
+  Future<void> initPlatformState() async {
+    AmplifyAPI apiPlugin = AmplifyAPI();
+
+    await amplify.addPlugin(apiPlugins: [apiPlugin]);
+
+    // Configure
+    await amplify.configure(amplifyconfig);
+    setState(() {
+      _isAmplifyConfigured = true;
```

Comment:
Seems like, if you're going to keep this state boolean, might as well do it in the façade so that people can ask like `Amplify.isConfigured()`.

Reply:
Makes sense. We can consider adding this change in the current facade PR.